packages/teams-js/test/private/videoEffectsEx.spec.ts

```diff
@@ -234,6 +237,42 @@ describe('videoEffectsEx', () => {
         expect(message?.args?.[1]).toEqual(videoEffectsEx.ErrorLevel.Warn);
       });
 
+      it('should send notifyError when frameProcessed event time outs', async () => {
+        await utils.initializeWithContext(FrameContexts.sidePanel);
+        const errorMessage = `Frame not processed in ${videoEffectsEx.frameProcessInterval}ms`;
```

Comment:
>Frame not processed in ${videoEffectsEx.frameProcessInterval}ms [](http://example.com/codeflow?start=30&length=63)

Can this error message be exported from videoEffectsEx so it doesn't have to be duplicated here?

Reply:
I think it's inappropriate to export an error msg from videoEffectsEx. There is no another file that needs to use this msg.

Reply:
I'm not sure what you mean -- the test file is another file that needs to use the message, right? Wouldn't that be barrel than duplicating the message and causing future maintenance issues where anyone who updates the string has to remember to update it twice?